### PR TITLE
[FSDE-24708] Include Save view component in New Vue documentation

### PR DESCRIPTION
**Pull request by josecarlosgonzalezv**

### CODE REVIEW

**Review by jllr — APPROVED** [review]

File: src/documentation/package-lock.json [hunk 1]
```diff
@@ -2947,7 +2948,7 @@
     "alphanum-sort": {
       "version": "1.0.2",
       "resolved": "https://registry.npmjs.org/alphanum-sort/-/alphanum-sort-1.0.2.tgz",
-      "integrity": "sha512-0FcBfdcmaumGPQ0qPn7Q5qTgz/ooXgIyp1rf8ik5bGX8mpE2YHjC0P/eyQvxu1GURYQgq9ozf2mteQ5ZD9YiyQ=="
+      "integrity": "sha1-l6ERlkmyEa0zaR2fn0hqjsn74KM="
```
Comment (janibolkvadze) [inline src/documentation/package-lock.json:2951]: Package-lock file should not be changed here.

File: src/documentation/package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
     "generate-production": "DOCS_ENV=production nuxt generate"
   },
   "dependencies": {
-    "@centurylink/chi-vue": "1.3.5",
+    "@centurylink/chi-vue": "1.5.0",
```
Comment (janibolkvadze) [inline src/documentation/package.json:14]: 1.5.1

File: src/documentation/pages/components-wip/toolbar/_accessibility.vue [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <template lang="pug">
-  <AccessibilityComingSoon />
+  AccessibilityComingSoon
```
Comment (janibolkvadze) [inline src/documentation/pages/components-wip/toolbar/_accessibility.vue:2]: `<AccessibilityComingSoon />`

File: src/documentation/pages/components-wip/toolbar/_properties.vue [hunk 4]
```diff
@@ -107,6 +107,93 @@
...
+                div
+                  code default
+              td
+                div To set the default.
+              td
+                div
+                  code boolean
+              td
+                div
+                  code true
```
Comment (janibolkvadze) [inline src/documentation/pages/components-wip/toolbar/_properties.vue:148]: Is this accurate default value?

File: src/documentation/pages/components-wip/toolbar/index.vue [hunk 5]
```diff
@@ -1,13 +1,13 @@
 <template lang="pug">
   div
-    <TitleBar title="Toolbar" description="Toolbar provides the ability of performing actions such as searching for elements and filtering complex structures." :tabs="pageTabs" />
+    TitleBar(title="Toolbar" description="Toolbar provides the ability of performing actions such as searching for elements and filtering complex structures." :tabs="pageTabs")
     .chi-grid__container.-pt--3
       .chi-tabs-panel.-active#examples
-        <Examples />
+        Examples
```
Comment (janibolkvadze) [inline src/documentation/pages/components-wip/toolbar/index.vue:6]: `<Examples />`

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

**Review by janibolkvadze — APPROVED** [review]

**Pull request merged**